src/ddmd/utils.d

```diff
@@ -92,6 +92,7 @@ extern (C++) void writeFile(Loc loc, File* f)
  */
 extern (C++) void ensurePathToNameExists(Loc loc, const(char)* name)
 {
+    import core.stdc.stdio;
```

Comment:
Why was this added? A deprecation message about imports? Certainly, you didn't add anything here which uses the import.

Reply:
Leftover from printf debugging.